Description: Upstream changes introduced in version 0.0.2-72yavdr1~precise
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 vdr-plugin-solitaire (0.0.2-72yavdr1~precise) precise; urgency=medium
 .
   * Standards-Version: 3.9.2
   * Build-depend on vdr-dev >= 1.7.27
   * Switched to debhelper 7 and dropped cdbs
   * Added i18n.patch
 .
 The person named in the Author field signed this changelog entry.
Author: Holger Schvestka <hotzenplotz5@gmx.de>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- vdr-plugin-solitaire-0.0.2.orig/game.c
+++ vdr-plugin-solitaire-0.0.2/game.c
@@ -4,7 +4,6 @@
 #include "cards.h"
 #include "osd.h"
 #include "cursor.h"
-#include "i18n.h"
 #include <math.h>
 
 cSolitaireGame::cSolitaireGame(void): cOsdObject(true) {
